wap_plugin.py

```diff
@@ -496,13 +497,20 @@ def downloadCroppedRaster(self):
 
         rast_url = self.api_manag.query_crop_raster(params)
 
-        self.file_manag.download_raster(rast_url)
+        rast_directory = self.dlg.downloadFolderExplorer.filePath()
+        self.file_manag.download_raster(rast_url, rast_directory)
         
         self.dlg.progressBar.setValue(100)
         self.dlg.progressLabel.setText ('Raster Download Complete')
         
         self.listRasterMemory()
 
+    def updateRasterFolder(self):
```

Comment:
I think we have to standardize function names to either `underscore_separated` or `camelCase`.